libcontainer/selinux/selinux.go

```diff
@@ -158,12 +158,16 @@ func Setfilecon(path string, scon string) error {
 // Getfilecon returns the SELinux label for this path or returns an error.
 func Getfilecon(path string) (string, error) {
 	con, err := system.Lgetxattr(path, xattrNameSelinux)
-
+	if err != nil {
+		return "", err
+	}
 	// Trim the NUL byte at the end of the byte buffer, if present.
-	if con[len(con)-1] == '\x00' {
-		con = con[:len(con)-1]
+	if len(con) > 0 {
+		if con[len(con)-1] == '\x00' {
```

Comment:
It's minor, but I think we should change it to `if len(con) > 0 && con[len(con)-1] == '\x00'`.